vsintegration/src/FSharp.Editor/LanguageService/SingleFileWorkspaceMap.fs

```diff
@@ -74,7 +83,47 @@ type internal SingleFileWorkspaceMap(workspace: VisualStudioWorkspace,
 
         member _.OnAfterFirstDocumentLock(_, _, _, _) = VSConstants.E_NOTIMPL
 
-        member _.OnAfterSave(_) = VSConstants.E_NOTIMPL
+        member _.OnAfterSave(docCookie) = 
```

Comment:
If you don't want to use the running doc table APIs, there is a command in the editor commanding system you can intercept instead: https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.text.editor.commanding.commands.savecommandargs?view=visualstudiosdk-2019

Reply:
Not sure how to use the commanding system. Is there any benefit to it versus just using RDT?  (other than not using RDT)

Reply:
Well, it'd mean you can fix all the bugs in https://github.com/dotnet/fsharp/pull/11460#discussion_r621570322 easily!

Reply:
Ok, I looked at Roslyn to see how it handles `SaveCommandArgs` and now I'm using the commanding system instead of RDT - and yes, it is a bit nicer :)